chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml

```diff
@@ -61,17 +61,20 @@
 
     - name: Patch the engine UID
       template:
-        src:  /chaoslib/litmus/disk_fill/disk_fill_ds.j2
-        dest: /chaoslib/litmus/disk_fill/disk_fill_ds.yml
-      
-    - name: Create the DaemonSet for Disk-Fill
+        src:  /chaoslib/litmus/disk_fill/disk_fill_job.j2
+        dest: /chaoslib/litmus/disk_fill/disk_fill_job.yml
+      vars:
+        chaosUID: "{{ chaos_uid }}"
```

Comment:
Can we directly use `chaos_uid` variable inside jinja template as it is globally available? pls refer any other experiments